src/main/java/com/iabtcf/utils/BitSetIntIterable.java

```diff
@@ -60,6 +61,14 @@ public boolean containsAll(int... source) {
         return true;
     }
 
+    @Override
+    public boolean containsAny(int... source) {
+        return Arrays
+                .stream(source)
+                .filter(value -> value < bs.size())
+                .anyMatch(this::contains);
+    }
+
```

Comment:
I believe that this method could go up as a default method in the interface.

Reply:
Unfortunately because of  using `bs.size()` it's not possible